server/src/main/java/org/elasticsearch/indices/recovery/RecoveryFileChunkRequest.java

```diff
@@ -64,7 +66,7 @@ public RecoveryFileChunkRequest(long recoveryId, final long requestSeqNo, ShardI
         this.shardId = shardId;
         this.metadata = metadata;
         this.position = position;
-        this.content = content;
+        this.content = ReleasableBytesReference.wrap(content);
```

Comment:
`content` is actually already a `ReleasableBytesReference` (in prod code at least)

Reply:
neat :)